elide-core/src/test/java/com/yahoo/elide/audit/LogMessageTest.java

```diff
@@ -73,15 +73,15 @@ public String getName() {
     @Test
     public void verifyOpaqueUserExpressions() {
         final String[] expressions = { "${opaqueUser.name}", "${opaqueUser.name}" };
-        final LogMessage message = new LogMessage("{0} {1}", childRecord, expressions, 1, Optional.empty());
+        final LogMessageImpl message = new LogMessageImpl("{0} {1}", childRecord, expressions, 1, Optional.empty());
```

Comment:
Can we rename the file to `LogMessageImplTest.java`?

Reply:
:+1: